documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_3_Securing_the_Platform.md

```diff
@@ -0,0 +1,59 @@
+* **Protecting confidentiality of data at rest**: Depending on the type of data, confidentiality of the data needs to be protected. It is typically achieved by encrypting data before persistently storing it, e.g. with disk encryption or by using a database that stores data only in encrypted form. Backups shall always be encrypted to prevent unauthorized access to them.
+* **Isolation of processes**: To reduce the attack surface, interactions between different applications on the system shall be kept to a minimum and each application shall only get the privileges it requires. This can be achieved by isolating and restricting applications (containers) using security mechanisms in the Linux kernel such as namespaces, cgroups, AppArmor, SELinux, Linux capabilities and seccomp.
+* **Tracking of events**: In case of security incidents, it is important to be able to reconstruct what happened on a connector. This is addressed by an integrity-protected audit log for all security-relevant events on the device. Logging should be performed on all layers: Within the platform up to the container runtime, e.g. by using a logging framework for Linux such as journald, and on the application layer. The latter can be done either by using a defined interface to the platform logging or by using an application-specific solution.
+* **Protecting utilized keys**: The cryptographic material (typically keys) used to achieve many of the requirements described above needs to be stored and used in a way that protects its confidentiality (for symmetric or private keys) or integrity (for public keys). Public keys used as root of trust for integrity and authenticity verification are typically provisioned onto the device during a secure initialization phase which is conducted in a secure environment. Private keys used by the device should be generated directly on the device and never directly displayed/provided to users or processes using them. Depending on the chosen Trust Level, the usage of hardware protection mechanisms such as, e.g., a HSM or TPM, is necessary.
```

Comment:
Is "Trust Level" a proper noun that needs to be capitalised?

Reply:
from my perspective yes, since we introduce this terminology in the certification context and always use it to refer to those Trust Levels in IDS Certification.